common/views/components/SearchFilters/SearchFiltersDesktop.js

```diff
@@ -10,9 +12,31 @@ import NumberInput from '@weco/common/views/components/NumberInput/NumberInput';
 // $FlowFixMe (tsx)
 import CheckboxRadio from '@weco/common/views/components/CheckboxRadio/CheckboxRadio';
 import NextLink from 'next/link';
+import dynamic from 'next/dynamic';
+import TogglesContext from '../TogglesContext/TogglesContext';
 import { type SearchFiltersSharedProps } from './SearchFilters';
 
-const CancelFilter = ({ text }: { text: string }) => {
+// $FlowFixMe (tsx)
+const ColorPicker = dynamic(import('../ColorPicker/ColorPicker'), {
```

Comment:
oh nice

Reply:
yeah, to be honest I don't really understand the intricacies of how/when dynamic imports get loaded (but it was because of the big bundle size that I had a go)